api-reference/node.mdx

```diff
@@ -223,7 +223,7 @@ const document = await metal.getOne("documentId-123");
 ```ts
 
 const documentIDs = ['document_id_123', 'document_id_456'];
-const result = await Metal.getManyDocuments(documentIDs);
+const result = await Metal.getMany(documentIDs);
```

Comment:
Ah did we rename this method or was this a typo from before?

Reply:
yes, it was a typo from before!